src/generic-provider-views/Browser.js

```diff
@@ -60,6 +60,15 @@ module.exports = (props) => {
           title: props.title
         })}
       </div>
+      <button class="UppyButton--circular UppyButton--blue Browser-doneBtn"
+              type="button"
+              aria-label="Done picking files"
+              title="Done picking files"
```

Comment:
can we localise these? not sure if we have easy access to a translator instance from `generic-provider-views`

Reply:
Yeah, that’s the problem, where would the strings come from? Dashboard makes the most sense, and at this point we basically have to admit that “target: Dashboard” will be the only way to use GoogleDrive and Instagram.